### DISCUSSION TITLE
カップラーメンタイマの件

### DISCUSSION THREAD

**Discussion by tsukada-shigeki** [toppers/users-forum#75] [TOPPERS Seminar]
竹内様

お世話になります。
base3/program3/CupRamenTimerをmy_programにコピー。ビルド後、実行しましたが動作致しません。お手数ですが添付ファイルをご確認の上推定原因をご教示願います。
以上宜しくお願い致します。
塚田
[CupRamenTimer.zip](https://github.com/toppers/users-forum/files/8915345/CupRamenTimer.zip)

**Comment by Ryohsuke-Takeuchi** [comment]
塚田様から送られたCupRamenTimer中のCupRamenTimer.srecを、TeraTermでダウンロードして実行したところ
写真のように、問題なく動いていますが、
動かいないとは、どういう状態でしょうか？
基礎３のmy_programが動いていれば、ハードの問題でもないように思われますが？

![IMG_20220616_174647](https://user-images.githubusercontent.com/91311953/174032228-bf9ce730-b759-438a-acd3-c8f9ab7da906.jpg)

**Comment by Ryohsuke-Takeuchi** [comment]
原因がわかりました。
当方で、塚田様のソースをclearした後、ビルドして実行し、ちゃんと動くことを確認しました。
その後で、塚田様のCupRemanTimer.srecをダウンロードして実行したのですが
上記の写真の通り、ダウンロードアドレスが0x0800000となっています。
マップファイルを作って確認したところ、ROM実行になっていますので、FLASHに書き込んで実行してください。
よろしくお願いいたします。
![image](https://user-images.githubusercontent.com/91311953/174034854-cf82448e-0a80-486c-8783-03108e950070.png)

**Reply by tsukada-shigeki** [reply]
竹内様

お世話になります。
ご指摘有難うございました。
正常動作確認できました（添付ファイル参照願います）。
![カップラーメンタイマ](https://user-images.githubusercontent.com/106538497/174083743-0b5ec070-2bf1-44c7-be23-ddf69ab8ef99.jpg)
